.github/workflows/release-brew.yaml

```diff
@@ -3,21 +3,118 @@ on:
+          username: ${{ env.BOT_USER }}
+
       - name: Create homebrew PR
         run: |
+          brew tap ${{ env.TAP }}
           brew update-reset
-          brew tap aws/tap
-          brew bump-formula-pr --tag "$RELEASE_TAG" --revision "$GITHUB_SHA" aws/tap/cbmc-viewer
+          brew bump-formula-pr --tag "${{ env.RELEASE_TAG }}" --revision "$GITHUB_SHA" ${{ env.TAP }}/${{ env.FORMULA }}
         env:
           HOMEBREW_GITHUB_API_TOKEN: ${{ secrets.RELEASE_CI_ACCESS_TOKEN }}
+
+  build-bottle:
+    needs: homebrew-pr
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, macos-10.15]
+    runs-on: ${{ matrix.os }}
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+
+      - name: Checkout PR
+        run: |
+          brew tap ${{ env.TAP }}
+          brew update-reset
+          cd $(brew --repo ${{ env.TAP }})
+          git remote add fork-repo ${{ env.FORK_REPO }}
+          git fetch fork-repo
+          git checkout -B bump-${{ env.FORMULA }}-${{ env.VERSION }} fork-repo/bump-${{ env.FORMULA }}-${{ env.VERSION }}
+
+      - name: Tap Syntax
+        run: |
+          brew audit --online --git --skip-style ${{ env.TAP }}/${{ env.FORMULA }}
+          brew style ${{ env.TAP }}/${{ env.FORMULA }}
+
+      - name: Build bottle
+        run: |
+          brew test-bot --tap ${{ env.TAP }} --testing-formulae ${{ env.TAP }}/${{ env.FORMULA }} --only-formulae --root-url=https://github.com/$GITHUB_REPOSITORY/releases/download/${{ env.RELEASE_TAG }}
+
+      - name: Get Package Path
+        id: get_package_path
+        run: |
+          bottle_name="$(ls *.tar.gz)"
+          echo "::set-output name=bottle::$bottle_name"
+
+      - name: Get File Name
+        id: get_file_name
+        run: |
+          brew install jq
+          file_name="$(cat *.json | jq -r '."${{ env.TAP }}/${{ env.FORMULA }}".bottle.tags[].filename')"
+          echo "::set-output name=file_name::$file_name"
+
+      - name: Upload bottles as artifact
+        if: always()
+        uses: actions/upload-artifact@main
+        with:
+          name: bottles
+          path: '*.bottle.*'
+
+      - name: Upload release binary
+        uses: actions/upload-release-asset@v1.0.2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ github.event.release.upload_url }}
+          asset_path: ${{ steps.get_package_path.outputs.bottle }}
+          asset_name: ${{ steps.get_file_name.outputs.file_name }}
+          asset_content_type: application/x-gzip
+
+  update-pr:
+    needs: build-bottle
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.RELEASE_CI_ACCESS_TOKEN }}
+    steps:
+    - uses: actions/download-artifact@v3
+      with:
+        name: bottles
+
+    - name: Configure git user name and email
```

Comment:
Here an above you configure only username and not email

Reply:
This action automatically sets up an email such as  - 
```
    - name: Configure git user name and email
      uses: Homebrew/actions/git-user-config@master
      with:
        username: aws-viewer-for-cbmc-release-ci
```
will run two following commands -
```
/usr/bin/git config --global user.name aws-viewer-for-cbmc-release-ci
/usr/bin/git config --global user.email 100316598+aws-viewer-for-cbmc-release-ci@users.noreply.github.com
```

Do you have a specific email in mind to set this to?

Reply:
I think the viewer address used for pypi would make sense.